### PR TITLE
added WIT definition

### PR DESCRIPTION
**Pull request by jsalowey**
Resolves issues #34

### CODE REVIEW

File: draft-ietf-wimse-arch.md [hunk 1]
```diff
@@ -75,6 +75,10 @@ Identity proxy is an intermediary that can inspect, replace or augment workload
 
 Attestation is the function through which a task verifies the identity of a separate Workload. (TBD: sync definition with reference RaTS architecture)
 
+* Workload Identity Token
```
Comment (yaroslavros) [inline draft-ietf-wimse-arch.md:78]: Maybe call it WIMSE token?

File: draft-ietf-wimse-arch.md [hunk 2]
```diff
@@ -75,6 +75,10 @@ Identity proxy is an intermediary that can inspect, replace or augment workload
...
+
+An authentication token that contains a workload identifier used for service to service authentication. The token is bound to a cryptographic key and requires that the presenter provide proof of possession of the secret key material. This token is further defined in {{?I-D.ietf-wimse-s2s-protocol}}
```
Comment (yaroslavros) [inline draft-ietf-wimse-arch.md:80]: Let's drop first "authentication". That is,

A token that contains a workload identifier used for service to service authentication. It is bound to a cryptographic key and requires that the presenter provide proof of possession of the secret key material. This token is further defined in {{?I-D.ietf-wimse-s2s-protocol}}

File: draft-ietf-wimse-arch.md (hunk 1 above)
Comment (jsalowey) [reply draft-ietf-wimse-arch.md:78]: This is what it is called in the S2S spec,  we would need to change the name there.

File: draft-ietf-wimse-arch.md (hunk 2 above)
Comment (jsalowey) [reply draft-ietf-wimse-arch.md]: Updated

**Pull request merged**